content/en/code_analysis/faq/_index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Code Analysis Frequently Asked Questions
+description: What are the frequently asked questions about Datadog Code Analysis
+further_reading:
+- link: "/code_analysis/"
+  tag: "Documentation"
+  text: "Learn about Code Analysis"
+- link: "/code_analysis/static_analysis/"
+  tag: "Documentation"
+  text: "Learn about Static Analysis"
+- link: "/code_analysis/software_composition_analysis/"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+---
+
+## Common questions
+
+### What are Datadog-hosted scans?
+
+When using a Datadog-hosted scan, Datadog scans your code within the Datadog infrastructure. Datadog acts
+as a CI pipeline: it clones your code, runs the analyzer, and uploads the results for you.
+
+With Datadog-hosted scans, you do not need to do the heavy lifting and configure any CI pipeline to run the analyzer.
```

Comment:
The benefit of Datadog-hosted scans is that no configuration is needed in your CI pipeline(s) to use Code Analysis.